bin/sdk-cli.js

```diff
@@ -24,11 +24,11 @@ const buildBlock = argv => {
 };
 
 yargs
-	.scriptName( 'calypso-gutenberg-sdk' )
+	.scriptName( 'calypso-sdk' )
```

Comment:
This needs to match the name in `package.json`'s `bin` entry. It gets printed in help output in place of `$0`